bin/broadside

```diff
@@ -89,6 +89,15 @@ command :deploy do |d|
     add_shared_deploy_configs(subcmd)
   end
 
+  d.desc 'Creates an initial task definition and service, then runs a series of bootstrap commands.'
```

Comment:
Ahhh I didn't see that. Doesn't it make sense to run `bootstrap` for a specific target?